--------------------------------------------------------------
Begin Slurm Prologue Sat Sep  7 16:31:33 CDT 2024 1725744693
Job ID:		1610296
Username:	l.peiwang
Partition:	tier2_gpu
End Slurm Prologue Sat Sep  7 16:31:33 CDT 2024 1725744693
--------------------------------------------------------------
2024-09-07 16:31:33.655884: W tensorflow/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libcudart.so.11.0'; dlerror: libcudart.so.11.0: cannot open shared object file: No such file or directory
2024-09-07 16:31:33.655923: I tensorflow/stream_executor/cuda/cudart_stub.cc:29] Ignore above cudart dlerror if you do not have a GPU set up on your machine.
/home/l.peiwang/liuenv/lib/python3.10/site-packages/tensorflow_addons/utils/tfa_eol_msg.py:23: UserWarning: 

TensorFlow Addons (TFA) has ended development and introduction of new features.
TFA has entered a minimal maintenance and release mode until a planned end of life in May 2024.
Please modify downstream libraries to take dependencies from other repositories in our TensorFlow community (e.g. Keras, Keras-CV, and Keras-NLP). 

For more information see: https://github.com/tensorflow/addons/issues/2807 

  warnings.warn(
/home/l.peiwang/liuenv/lib/python3.10/site-packages/tensorflow_addons/utils/ensure_tf_install.py:53: UserWarning: Tensorflow Addons supports using Python ops for all Tensorflow versions above or equal to 2.11.0 and strictly below 2.14.0 (nightly versions are not supported). 
 The versions of TensorFlow you are currently using is 2.8.4 and is not supported. 
Some things might work, some things might not.
If you were to encounter a bug, do not file an issue.
If you want to make sure you're using a tested and supported configuration, either change the TensorFlow version or the TensorFlow Addons's version. 
You can find the compatibility matrix in TensorFlow Addon's readme:
https://github.com/tensorflow/addons
  warnings.warn(
/home/l.peiwang/liuenv/lib/python3.10/site-packages/nilearn/input_data/__init__.py:23: FutureWarning: The import path 'nilearn.input_data' is deprecated in version 0.9. Importing from 'nilearn.input_data' will be possible at least until release 0.13.0. Please import from 'nilearn.maskers' instead.
  warnings.warn(message, FutureWarning)
/home/l.peiwang/liuenv/lib/python3.10/site-packages/nilearn/maskers/nifti_masker.py:98: UserWarning: imgs are being resampled to the mask_img resolution. This process is memory intensive. You might want to provide a target_affine that is equal to the affine of the imgs or resample the mask beforehand to save memory and computation time.
  warnings.warn(
/home/l.peiwang/liuenv/lib/python3.10/site-packages/nilearn/maskers/nifti_masker.py:98: UserWarning: imgs are being resampled to the mask_img resolution. This process is memory intensive. You might want to provide a target_affine that is equal to the affine of the imgs or resample the mask beforehand to save memory and computation time.
  warnings.warn(
/home/l.peiwang/liuenv/lib/python3.10/site-packages/nilearn/maskers/nifti_masker.py:98: UserWarning: imgs are being resampled to the mask_img resolution. This process is memory intensive. You might want to provide a target_affine that is equal to the affine of the imgs or resample the mask beforehand to save memory and computation time.
  warnings.warn(
/home/l.peiwang/liuenv/lib/python3.10/site-packages/nilearn/maskers/nifti_masker.py:98: UserWarning: imgs are being resampled to the mask_img resolution. This process is memory intensive. You might want to provide a target_affine that is equal to the affine of the imgs or resample the mask beforehand to save memory and computation time.
  warnings.warn(
/home/l.peiwang/liuenv/lib/python3.10/site-packages/nilearn/maskers/nifti_masker.py:98: UserWarning: imgs are being resampled to the mask_img resolution. This process is memory intensive. You might want to provide a target_affine that is equal to the affine of the imgs or resample the mask beforehand to save memory and computation time.
  warnings.warn(
/home/l.peiwang/liuenv/lib/python3.10/site-packages/nilearn/maskers/nifti_masker.py:98: UserWarning: imgs are being resampled to the mask_img resolution. This process is memory intensive. You might want to provide a target_affine that is equal to the affine of the imgs or resample the mask beforehand to save memory and computation time.
  warnings.warn(
/home/l.peiwang/liuenv/lib/python3.10/site-packages/nilearn/maskers/nifti_masker.py:98: UserWarning: imgs are being resampled to the mask_img resolution. This process is memory intensive. You might want to provide a target_affine that is equal to the affine of the imgs or resample the mask beforehand to save memory and computation time.
  warnings.warn(
/home/l.peiwang/liuenv/lib/python3.10/site-packages/nilearn/maskers/nifti_masker.py:98: UserWarning: imgs are being resampled to the mask_img resolution. This process is memory intensive. You might want to provide a target_affine that is equal to the affine of the imgs or resample the mask beforehand to save memory and computation time.
  warnings.warn(
/home/l.peiwang/liuenv/lib/python3.10/site-packages/nilearn/maskers/nifti_masker.py:98: UserWarning: imgs are being resampled to the mask_img resolution. This process is memory intensive. You might want to provide a target_affine that is equal to the affine of the imgs or resample the mask beforehand to save memory and computation time.
  warnings.warn(
/home/l.peiwang/liuenv/lib/python3.10/site-packages/nilearn/maskers/nifti_masker.py:98: UserWarning: imgs are being resampled to the mask_img resolution. This process is memory intensive. You might want to provide a target_affine that is equal to the affine of the imgs or resample the mask beforehand to save memory and computation time.
  warnings.warn(
Number of CN subjects:
263
Number of PCN subjects:
140
Number of MCI subjects:
458
Number of Dementia subjects:
151
lenth of dataset: 
414
Traceback (most recent call last):
  File "/ceph/chpc/home/l.peiwang/Modality_Comparison_AD/deep_learning.py", line 175, in <module>
    train_model(X, Y, class_weights)
  File "/ceph/chpc/home/l.peiwang/Modality_Comparison_AD/deep_learning.py", line 124, in train_model
    X_train_augmented = np.array([augment_data(X[i]) for i in train_idx])
  File "/ceph/chpc/home/l.peiwang/Modality_Comparison_AD/deep_learning.py", line 124, in <listcomp>
    X_train_augmented = np.array([augment_data(X[i]) for i in train_idx])
  File "/ceph/chpc/home/l.peiwang/Modality_Comparison_AD/deep_learning.py", line 109, in augment_data
    augmented_image = np.flip(augmented_image, axis=axis)
  File "/home/l.peiwang/liuenv/lib/python3.10/site-packages/numpy/lib/function_base.py", line 340, in flip
    axis = _nx.normalize_axis_tuple(axis, m.ndim)
  File "/home/l.peiwang/liuenv/lib/python3.10/site-packages/numpy/core/numeric.py", line 1380, in normalize_axis_tuple
    axis = tuple([normalize_axis_index(ax, ndim, argname) for ax in axis])
  File "/home/l.peiwang/liuenv/lib/python3.10/site-packages/numpy/core/numeric.py", line 1380, in <listcomp>
    axis = tuple([normalize_axis_index(ax, ndim, argname) for ax in axis])
numpy.exceptions.AxisError: axis 1 is out of bounds for array of dimension 1
--------------------------------------------------------------
Begin Slurm Epilogue Sat Sep  7 16:34:51 CDT 2024 1725744891
Name                : early-min
User                : l.peiwang
Partition           : tier2_gpu
Nodes               : gpu08
Cores               : 1
State               : COMPLETED
Submit              : 2024-09-07T16:31:29
Start               : 2024-09-07T16:31:33
End                 : 2024-09-07T16:34:48
Reserved Walltime   : 5-10:50:00
Used Walltime       :   00:03:15
Used CPU Time       :   00:02:56
% User (Computation): 96.90%
% System (I/O)      :  3.10%
Mem Reserved        : 100000M
Max Mem Used        : 626.84M (657285120.0)
Max Disk Write      : 10.24K (10485.76)
Max Disk Read       : 8.62G (9259901255.68)
Max-Mem-Used Node   : gpu08
Max-Disk-Write Node : gpu08
Max-Disk-Read Node  : gpu08
Tesla T4
Error: Unable to retrieve job statistics. Return: No data is available.
End Slurm Epilogue Sat Sep  7 16:34:51 CDT 2024 1725744891
--------------------------------------------------------------
